README.md

```diff
@@ -33,15 +33,15 @@ require 'vendor/autoload.php';
 use AltchaOrg\Altcha\ChallengeOptions;
 use AltchaOrg\Altcha\Altcha;
 
-$hmacKey = 'secret hmac key';
+$altcha = new Altcha('secret hmac key');
 
 // Create a new challenge
-$options = new ChallengeOptions([
-    'hmacKey'   => $hmacKey,
-    'maxNumber' => 50000, // the maximum random number
+$options = new ChallengeOptions(
+    maxNumber: 50000, // the maximum random number
+    expires: (new \DateTimeImmutable())->add(new \DateInterval('PT10S')),
 ]);
```

Comment:
Named arguments in PHP make it a lot easier to skip arguments with default values.